index.js

```diff
@@ -0,0 +1,19 @@
+const { Client } = require('discord.js');
+const { config } = require('dotenv');
+const { Handler } = require('discord-slash-command-handler');
+// Init .env configuration
+config();
+
+const eternalClient = new Client({ intents: ["GUILDS", "GUILD_MESSAGES"] });
```

Comment:
Nie lepiej w tym przypadku użyć bitsów?

Reply:
Musiałem wrócić do wersji 13.x discord.js ponieważ handler nie wspiera 14

Reply:
To lepiej zrobić własny handler niż wracać do starej wersji discord.js?

Reply:
Sądzę że to nie ma sensu, choć nie mam pewności czy czasem na 14.x nie zadziała. Będę testował lecz pierw tutaj chociaż te podstawkę pod 13.x zrobiłem

Reply:
Pare minut i masz zrobiony slash command buider, chciałbym wykorzystać modale i inne ficzery z v14

Reply:
Spoko to to ogarnę póżniej